pkgs/misc/emulators/caprice32/default.nix

```diff
@@ -3,27 +3,31 @@
 stdenv.mkDerivation rec {
 
   pname = "caprice32";
-  version = "4.5.0";
+  version = "4.6.0";
 
   src = fetchFromGitHub {
     repo = "caprice32";
     rev = "v${version}";
     owner = "ColinPitrat";
-    sha256 = "056vrf5yq1574g93ix8hnjqqbdqza3qcjv0f8rvpsslqcbizma9y";
+    sha256 = "0hng5krwgc1h9bz1xlkp2hwnvas965nd7sb3z9mb2m6x9ghxlacz";
   };
 
-  postPatch = "substituteInPlace cap32.cfg --replace /usr/local $out";
   nativeBuildInputs = [ pkgconfig ];
   buildInputs = [ libpng SDL freetype zlib ];
 
-  #fix GIT_HASH avoid depend on git 
-  makeFlags = [ "GIT_HASH=${src.rev}" "DESTDIR=$(out)" "prefix=/"];
+  makeFlags = [
+    "APP_PATH=${placeholder ''out''}/share/caprice32/"
+    "RELEASE=1"
+    #"WITH_IPF=0"
```

Comment:
If this doesn’t need to be there, delete it.

Reply:
was a reminder to not try to enable it to 0 since option is not functional, anyway i'll remove.